### PR TITLE
floating point PCM support

### PR DESCRIPTION
**Pull request by dvdli**

### CODE REVIEW

File: src/pcm.c [hunk 1]
```diff
@@ -282,6 +282,11 @@ static unsigned int pcm_format_to_alsa(enum pcm_format format)
         return SNDRV_PCM_FORMAT_S32_LE;
     case PCM_FORMAT_S32_BE:
         return SNDRV_PCM_FORMAT_S32_BE;
+
+    case PCM_FORMAT_FLOAT_LE:
```
Comment (dbaluta) [inline src/pcm.c:286]: Do you really want the new line here?

Comment (dvdli) [reply src/pcm.c:286]: Yes.

You would find that others big and little endian formats are also in groups and have a new line between each group.

Comment (dbaluta) [reply src/pcm.c:286]: Ok, great. Thanks!

**Comment by dvdli** [comment]
@tay10r @gkasten @simonrules
Could you please help to review this pull request?

**Review by tay10r — APPROVED** [review]
Everything looks good to me! Good to have FP support now. Nice work!

**Pull request merged**